ansible/roles/openshift_outbound_tcp_logging/tasks/main.yml

```diff
@@ -1,5 +1,11 @@
 ---
 - set_fact:
+    outbound_filtering_chain:
+    - chain: OPENSHIFT-OUTPUT-FILTERING
+      line:  ":OPENSHIFT-OUTPUT-FILTERING - [0:0]"
+      after: "filter$"
```

Comment:
I'd prefer something that more tightly matches so that we're not matching something from another table that happens to end with filter. Maybe something like:

`after: "^\\*filter$"`